tracing/src/main/java/com/palantir/tracing/Tracers.java

```diff
@@ -104,52 +104,74 @@ public static Runnable wrap(Runnable delegate) {
         return new TracingAwareRunnable(delegate);
     }
 
+    /**
+     * Like {@link #wrapWithNewTrace(String, ExecutorService)}, but with a default initial span operation.
+     */
+    public static ExecutorService wrapWithNewTrace(ExecutorService executorService) {
+        return wrapWithNewTrace(ROOT_SPAN_OPERATION, executorService);
+    }
+
     /**
      * Wraps the provided executor service to make submitted tasks traceable with a fresh {@link Trace trace}
-     * for each execution, see {@link #wrapWithNewTrace(ExecutorService)}. This method should not be used to
+     * for each execution, see {@link #wrapWithNewTrace(String, ExecutorService)}. This method should not be used to
      * wrap a ScheduledExecutorService that has already been {@link #wrap(ExecutorService) wrapped}. If this is
      * done, a new trace will be generated for each execution, effectively bypassing the intent of the previous
      * wrapping.
      */
-    public static ExecutorService wrapWithNewTrace(ExecutorService executorService) {
+    public static ExecutorService wrapWithNewTrace(String operation, ExecutorService executorService) {
```

Comment:
This will appear unredacted in Telemetry, so what do you think about another beloved `@CompileTimeConstant` to enforce that people don't accidentally exfil usernames or something?

Reply:
This would make it impossible to use thread pool name patterns as the operation when created a wrapped executor.

Reply:
I don't think this is the operation we'd use for thread pools, we'd need to use the DeferredTracer + overload Tracers.wrap(ExecutorService) which can be implemented in a separate PR

Reply:
We use this method when building scheduled executor services with new traces.